tests/e2e/ctl_v3_auth_cluster_test.go

```diff
@@ -0,0 +1,111 @@
+package e2e
```

Comment:
nit: `Copyright 2022 The etcd Authors` section for any new files introduced in etcd code base. 

Not sure why CI test missed this static check.

Reply:
Added copyright header.